package/android/src/main/java/com/shopify/reactnative/skia/SkiaDomView.java

```diff
@@ -14,6 +14,7 @@ public SkiaDomView(Context context) {
         super(context);
         RNSkiaModule skiaModule = ((ReactContext) context).getNativeModule(RNSkiaModule.class);
         mHybridData = initHybrid(skiaModule.getSkiaManager());
+        this.init();
```

Comment:
@chrfalch here we create the SurfaceTexture in the constructor which is not good. After the first display, if the view become detached/reattached the TextureView can destroy the SurfaceTexture and recreate one by itself (which is the part we want to avoid).